bench/main.ml

```diff
@@ -44,33 +44,44 @@ let rec finds t count = function
       ignore (Index.find t k);
       finds t (count + 1) tl
 
-let run input =
+let print_json input file =
```

Comment:
you can maybe move this to `common`? as it is used in both `db_bench.ml` and `main.ml`